### PR TITLE
pwd, zsh: add Spanish translation

### PR DESCRIPTION
**Pull request by victorhck**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/common/zsh.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# zsh
+
+> Z SHell, un intérprete para la línea de comandos compatible con Bash.
+> Ver también `histexpand` para la expansión del historial.
```
Comment (navarroaxel) [inline pages.es/common/zsh.md:4]:
```suggestion
> Véase también `histexpand` para la expansión del historial.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Just one comment.

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @victorhck

**Review by CleanMachine1 — APPROVED** [review]
Good page!

:shipit:

**Pull request merged**